ledger/accountdb.go

```diff
@@ -749,6 +784,82 @@ func totalAccounts(ctx context.Context, tx *sql.Tx) (total uint64, err error) {
 	return
 }
 
+// reencodeAccounts reads all the accounts in the accountbase table, decode and reencode the account data.
+// if the account data is found to have a different encoding, it would update the encoded account on disk.
```

Comment:
The discrepancy in the encoding is expected, but only for accounts that were created before 2.0.5 and haven't been updated since. This function is not going to be called routinely but only once during the upgrade.
This step would align all the databases on the network to have the same encoding.